pycnite/linetable.py

```diff
@@ -241,14 +234,28 @@ def read_all(self):
         return ret
 
 
-class ExceptionTableReader(VarintReader):
+class ExceptionTableReader:
     """Read the exception table in 3.11+."""
 
     def __init__(self, code: types.CodeType311):
         self.table = code.co_exceptiontable
         self.pos = 0
         self.end_pos = len(self.table)
 
+    def _read_byte(self):
+        b = self.table[self.pos]
```

Comment:
Once in a blue moon, I wish Python let you do self.pos++ =)

Reply:
yeah :) the c code just did stuff like `*pos++`